index.js

```diff
@@ -40,6 +42,25 @@ function increaseSpecifityOfRule(rule, opts) {
 	}
 }
 
+/**
+ * Checks for the presence of enable/disable comments up to the previous rule.
+ * @param  {node} node 		PostCSS Node
+ * @return {Null|Boolean} Null if no matching comment; otherwise boolean (toggle status)
+ */
+function checkForEnableDisableComments(node) {
+	if (!node) return null;
+
+	for (var prev = node; prev = prev.prev(); ) {
```

Comment:
This for-loop variation with the `final-expression` as the `condition` threw me off a bit. 

Let's update this to the more familiar 3 part syntax,

```
for (var prev = node.prev(); prev !== prev.prev(); prev = prev.prev()) {
```